tests/bootstrap.php

```diff
@@ -23,10 +31,26 @@
  * Manually load the plugin being tested.
  */
 function _manually_load_plugin() {
-	require dirname( __DIR__ ) . '/wporg-gp-translation-events.php';
+	require_once _glotpress_path( '/tests/phpunit/includes/loader.php' );
+	require_once dirname( __DIR__ ) . '/wporg-gp-translation-events.php';
 }
-
 tests_add_filter( 'muplugins_loaded', '_manually_load_plugin' );
 
+global $wp_tests_options;
+$wp_tests_options['permalink_structure'] = '/%postname%';
+
 // Start up the WP testing environment.
 require "$_tests_dir/includes/bootstrap.php";
+
+// Require GlotPress test code.
+require_once _glotpress_path( '/tests/phpunit/lib/testcase.php' );
+require_once _glotpress_path( '/tests/phpunit/lib/testcase-route.php' );
+require_once _glotpress_path( '/tests/phpunit/lib/testcase-request.php' );
+
+function _install_glotpress() {
+	require_once ABSPATH . 'wp-admin/includes/upgrade.php';
+	require_once _glotpress_path( '/gp-includes/schema.php' );
+	require_once _glotpress_path( '/gp-includes/install-upgrade.php' );
+	gp_upgrade_db();
```

Comment:
```suggestion
	gp_upgrade_db();
	foreach ( explode( ';', file_get_contents( __DIR__ . '/../schema.sql') ) as $sql ) {
		if ( trim($sql) ) {
			$wpdb->query( $sql );
		}
	}

```

Reply:
Yep, agree. My intention was that it would work in all environments indeed (local, wp-env, CI), but I didn't consider that the table wasn't being created for local. I agree with the solution you proposed of creating the table in `bootstrap.php`, which will work for all environments, and is also consistent with what we're doing for glotpress (the glotpress tables get created in `bootstrap.php` too).

Will fix.